examples/chembl/chembl_graph_conv.py

```diff
@@ -29,7 +29,7 @@
 graph_model.add(dc.nn.GraphConv(128, n_feat, activation='relu'))
 graph_model.add(dc.nn.BatchNormalization(epsilon=1e-5, mode=1))
 graph_model.add(dc.nn.GraphPool())
-graph_model.add(dc.nn.GraphConv(128, 64, activation='relu'))
+graph_model.add(dc.nn.GraphConv(128, 128, activation='relu'))
```

Comment:
Thank you for your comment. I changed this layer size since this size caused the next error.

`ValueError: Dimensions must be equal, but are 128 and 64 for 'MatMul_21' (op: 'MatMul') with input shapes: [?,128], [64,128].`

Of course there were many ways to fix this problem, I took the same way with `example/benchmark.py`, recently I fixed.